server/auth/auth_migration_mapping.md

```diff
@@ -0,0 +1,519 @@
+# Auth Decorator Migration Mapping
```

Comment:
chore: we should not commit these into the repository.

However, I like that analyses and this should be turned into an issue or a tech debt epic description.